extensions/visualizations/oppia-visualization-sorted-tiles.directive.html

```diff
@@ -3,8 +3,25 @@
   <hr>
 </div>
 <ol class="oppia-visualization-sorted-tiles">
-  <li class="oppia-visualization-sorted-tiles-tile" ng-repeat="datum in data">
-    <div class="oppia-visualization-sorted-tiles-tile-answer"><[datum.answer]></div>
-    <div class="oppia-visualization-sorted-tiles-tile-frequency"><[datum.frequency]> times</div>
+  <li tabindex="-1"
+      ng-repeat="datum in data | orderBy : -datum.frequency"
```

Comment:
does this scale with the number of options? Like maybe it becomes scrollable or something? I think it looks good with 2 options as you showed in the gif, but did you try with ~4-5 (which is typically the number of options I think).

Reply:
Also, should this also be the visualization for ItemSelectionInput?

Reply:
Hmm I don't think so, ItemSelectionInput seems to already work well with the current visualization.

Reply:
I've added logic to help make sure the content is always readable, here's a screencast for demonstration:
![screencast](https://user-images.githubusercontent.com/5094060/86524367-ead4ee00-be47-11ea-9bc2-85aff0652415.gif)

Reply:
Oooh I like this! Awesome thanks @brianrodri 😀

Reply:
Thanks to @swaat-on-git for the great design :)